Formula/python3.rb

```diff
@@ -107,6 +107,9 @@ def install
     ENV["PYTHONHOME"] = nil
     ENV["PYTHONPATH"] = nil
 
+    ver = version.to_s.slice(/(3\.\d)/) || File.read("configure.ac").slice(/PYTHON_VERSION, (3\.\d)/, 1)
```

Comment:
I think you could just do:
```
(buildpath/"configure.ac").read.slice(/PYTHON_VERSION, (3\.\d)/, 1)
```
unless there's times when that file isn't present? If so, it'd be best to use `build.head?`, `build.stable?` etc. rather than the `||`.